qgroundcontrol.pro

```diff
@@ -1403,3 +1403,39 @@ LinuxBuild {
 
     INSTALLS += target share_qgroundcontrol share_icons share_metainfo share_applications
 }
+
+
+#-------------------------------------------------------------------------------------
+#For 3D Viewer
+
+QT += \
+    quick3d \
```

Comment:
Instead of adding this here, you should add it to main place where all modules are included above this.